### PR TITLE
Documentation for Css Heading component

### PR DESCRIPTION
**Pull request by hcvdhaar**
- Added Storybook readme  component for `Heading`
- Added Readme for Heading

### CODE REVIEW

File: packages/css/src/heading/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Een gebruiker zou middels headings een goede indruk moeten krijgen van de content van een webpagina. Een heading zegt iets over de onderstaande content.
+
+## Toegankelijkheid
+
+Headings en de heading structuur zijn vooral belangrijk voor screen readers en zoekmachines.
+
+## Richtlijnen
+
+- Één `h1` tag per pagina die het hoofdonderwerp van de pagina beschrijft.
```
Comment (alimpens) [inline packages/css/src/heading/README.md:11]: Do we want to make this a rule? I don't think it's necessary for a11y, semantics, or SEO. At [onderzoek.amsterdam.nl](https://onderzoek.amsterdam.nl/) we use 3 `h1`s, corresponding to the `header`, `main` and `footer` sections of the page.

File: packages/css/src/heading/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- Een heading omschrijft de content eronder.
+- Een heading niet gebruiken voor opmaak en styling.
```
Comment (alimpens) [inline packages/css/src/heading/README.md:13]:
```suggestion
- Gebruik een heading niet voor opmaak en styling, maar om de structuur van de pagina weer te geven.
```

File: packages/css/src/heading/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+- Gebruik heading hierarchisch en sla niet een heading niveau over. Dus na een `h1` volgt een `h2` en dus niet een `h3`.
```
Comment (alimpens) [inline packages/css/src/heading/README.md:14]:
```suggestion
- Gebruik headings hierarchisch en sla geen heading niveaus over. Dus na een `h1` volgt een `h2` en dus niet een `h3`.
```

File: packages/css/src/heading/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
```
Comment (alimpens) [inline packages/css/src/heading/README.md:15]: Do we want to add the relevant WCAG rules here as well?

File: packages/css/src/heading/heading-css.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# CSS Heading
+
+Dit is een HTML tag die toegankelijkheid en standaarden faciliteert. Het gebruik is niet zo zeer voor opmaakdoeleinden maar dient meer als informatie structuring. Het is hierarchisch van aard. De beschikbare headings zijn: `h1`, `h2`, `h3`, `h4`, `h5` en `h6`.
```
Comment (alimpens) [inline packages/css/src/heading/heading-css.md:3]:
```suggestion
Gebruik de juiste heading HTML-elementen als je van deze classes gebruik maakt. De beschikbare elementen zijn: `h1`, `h2`, `h3`, `h4`, `h5` en `h6`. Let bij het kiezen van een heading niveau op de structuur van de pagina. Heading niveaus worden gebruikt om informatie te groeperen. De titel van de pagina krijgt een `h1`. De eerste subtitel een `h2`. De daarna volgende titels van secties die horen bij die subtitel krijgen een `h3`, enzovoort. Als je een nieuwe sectie begint, gebruik je weer een `h2`. Het is toegestaan om stijl en semantiek los van elkaar te trekken. Een `h1` kan bijvoorbeeld de class `amsterdam-heading-2` krijgen. 
```

File: documentation/component-docs-guidelines.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+# Documentation guidelines
+
+Some guidelines to follow for having a consistent way of documentating components.
+
+## Where to place the documentation files
+
+Generic documentation of components should be placed in the CSS component folder, `/packages/css/scr/{component-name}`. The main file (`README.md`) contains all the generic information about this component.
+Optionally you could have CSS specific documentation. This should be named after the CSS component with a suffix `-css`, e.g. `heading-css.md`.
```
[the remark this answers is not in the document]
Comment (hcvdhaar) [reply documentation/component-docs-guidelines.md:8]: I guess for now keep it as simple as possible. I would opt for having it as part of the generic description

File: packages/css/src/heading/README.md (hunk 4 above)
Comment (hcvdhaar) [reply packages/css/src/heading/README.md:15]: Good one

File: packages/css/src/heading/README.md (hunk 1 above)
Comment (hcvdhaar) [reply packages/css/src/heading/README.md:11]: So many opinions about this on the internet :) Let's leave it out. There are no WCAG rules against it.

File: packages/css/src/heading/README.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+## Richtlijnen
+
+- Een heading omschrijft de content eronder.
+- Gebruik een heading niet voor opmaak en styling, maar om de structuur van de pagina weer te geven.
+- Gebruik headings hierarchisch en sla geen heading niveaus over. Dus na een `h1` volgt een `h2` en dus niet een `h3`.
+
+## Relevante WCAG eisen
+
+- [WCAG 1.3.1](https://www.w3.org/WAI/WCAG21/quickref/#qr-content-structure-separation-programmatic):
+  Informatie, structuur en relaties die via presentatie worden overgebracht, programmatisch bepalen of in tekst aanbieden
```
Comment (alimpens) [inline packages/css/src/heading/README.md:14]: In my PR I chose to describe what part of the WCAG rules are relevant to that component, you've chosen to write a generic description. What do we prefer?

An advantage of a generic description is that they're reusable for other components. A disadvantage is that it's harder to figure out in what way the description relates to the component. What does the text above mean for the Heading component?

File: packages/css/src/heading/README.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+- [WCAG 2.4.1](https://www.w3.org/WAI/WCAG21/quickref/#qr-navigation-mechanisms-skip): De mogelijkheid aanbieden om inhoudsblokken over te slaan
+- [WCAG 2.4.6](https://www.w3.org/WAI/WCAG21/quickref/#headings-and-labels): Beschrijf met headings het doel of het onderwerp van de onderliggende content
+- [WCAG 2.4.10](https://www.w3.org/WAI/WCAG21/quickref/#qr-navigation-mechanisms-headings): Gebruik headings om de content te structureren
```
Comment (alimpens) [inline packages/css/src/heading/README.md:17]: These 2 are good, I think. They describe what you should do with headings

File: packages/css/src/heading/README.md (hunk 7 above)
Comment (hcvdhaar) [reply packages/css/src/heading/README.md]: Made it more specific for heading. I think we need to make it readable for humans. WCAG can be very technical en theoretical.

File: packages/css/src/heading/README.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+## Richtlijnen
+
+- Een heading omschrijft de content eronder.
+- Gebruik een heading niet voor opmaak en styling, maar om de structuur van de pagina weer te geven.
+- Gebruik headings hierarchisch en sla geen heading niveaus over. Dus na een `h1` volgt een `h2` en dus niet een `h3`.
+
+## Relevante WCAG eisen
+
+- [WCAG 1.3.1](https://www.w3.org/WAI/WCAG21/quickref/#qr-content-structure-separation-programmatic):
+  Een heading duiden met de daarvoor bedoelde html tag. `h1` - `h6`. Dit maakt het voor iedere gebruiker duidelijk hoe de pagina gestructeerd
```
Comment (alimpens) [inline packages/css/src/heading/README.md:14]:
```suggestion
Tekst die er uit ziet als een heading wordt ook door een computer herkend als een heading. Door gebruik te maken van het juiste heading niveau is voor alle gebruikers duidelijk hoe de pagina is gestructeerd.
```

File: packages/css/src/heading/README.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+- [WCAG 2.4.1](https://www.w3.org/WAI/WCAG21/quickref/#qr-navigation-mechanisms-skip): Een heading helpt een gebruiker eerder naar de hoofdinhoud te navigeren met het toetsenbord
```
Comment (alimpens) [inline packages/css/src/heading/README.md:15]:
```suggestion
- [WCAG 2.4.1](https://www.w3.org/WAI/WCAG21/quickref/#qr-navigation-mechanisms-skip): Headings kunnen gebruikt worden om naar een volgende sectie te navigeren met het toetsenbord
```

**Review by VincentSmedinga — APPROVED** [review]

**Pull request merged**